### DISCUSSION TITLE
Weird question,about the "ipfs-api-prelude" ?

### DISCUSSION THREAD

**Discussion by gcp-development** [ferristseng/rust-ipfs-api#128] [Q&A] [answered]
What is the propose of this prelude in the git repo ?
https://github.com/ferristseng/rust-ipfs-api/tree/master/ipfs-api-prelude

Cheers,

**Comment by ferristseng** [comment, marked as answer]
It's basically just shared code between each of the backends, e.g. trait definitions and common implementation code

**Reply by gcp-development** [reply]
Thank you for the answer.
